domain-models-runtime/src/main/java/org/folio/rest/persist/Criteria/Criteria.java

```diff
@@ -114,15 +114,15 @@ private void createFromSnippet() {
       from = new From();
       from.setSnippet(ARRAY_FROM_CLAUSE + "("
         + PostgresClient.DEFAULT_JSONB_FIELD_NAME + GET_JSON_FIELD + arrayField + ")");
-      from.setAsValue(field.get(0).replaceAll("^'|'$", "")); //remove ''
+      from.setAsValue(field.get(0).replace("'", "")); //remove ''
```

Comment:
Can you create a separate pull request (mentioning the same RMB-850 issue) for each file when the changes are completely unrelated?

Then we can review and merge them separately.

Reply:
Sure, i will create a different branch then & this one i will keep it for  reference . thanks for the update.

Reply:
Thanks, I see you've created multiple branches/pull requests for this.

Some files are in several of the new branches. This is probably not intended.